packages/mui-material/src/Select/Select.test.js

```diff
@@ -2,10 +2,10 @@ import * as React from 'react';
 import { expect } from 'chai';
 import { spy, stub } from 'sinon';
 import {
-  describeConformance,
-  ErrorBoundary,
   act,
   createRenderer,
+  describeConformance,
+  ErrorBoundary,
```

Comment:
```suggestion
  describeConformance,
  ErrorBoundary,
  act,
  createRenderer,
```

Revert some unrelated changes